client/my-sites/plugins/plugin-details.jsx

```diff
@@ -354,20 +377,43 @@ function CTA( {
 		isJetpack ||
 		isVip
 	);
+
 	return (
-		<Button
-			className="plugin-details__install-button"
-			onClick={ () =>
-				onClickInstallPlugin( {
-					dispatch,
-					selectedSite,
-					slug,
-					upgradeAndInstall: shouldUpgrade,
-				} )
-			}
-		>
-			{ shouldUpgrade ? translate( 'Upgrade and install' ) : translate( 'Install and activate' ) }
-		</Button>
+		<>
+			<Dialog
+				isVisible={ showEligibility }
+				title="Eligibility"
```

Comment:
Nitpick: Do you think having translated title would be better?  Something like below:
```
title={ translate('Eligibility') }
```

Reply:
Good catch!